Majasdarbs/04-majasdarbs/OlegsKozlovs/Portfolio/css/style.css

```diff
@@ -0,0 +1,78 @@
+/* HEADER START */
+
+header nav > ul {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;    
+    position: fixed;
+    left: 0;
+    top: 0;
+    background: rgb(2,0,36);
+    background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);
+    height: auto;
+    width: 100%;  
+}
+
+header nav ul > li {
+    width: 270px;
+    padding: 15px 50px;
+}
+
+header nav ul > li:hover {
+    background-color: rgb(132, 235, 235);
+}
+
+header a {
+    color: white;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+header a:hover {
+    color: rebeccapurple;
```

Comment:
You can try adding translate (to the selector without hover), to add smooth transition for style change.
basically for 
header a {
transition: color 0.3s ease-in;
}